#!/usr/bin/pyhton3
def raise_exception_msg(message=""):
    raise NameError(message)
